### PR TITLE
Update body.md

### PR DESCRIPTION
**Pull request by annaako**
here is the english page update

## Checklist before requesting a review

- [x] I have performed a self-review of my changes
- [x] I have updated both Swedish and English pages (if not motivate why)

### CODE REVIEW

**Comment by annaako** [comment]
here is the english update!

File: en/clubs/tag-monkeys/body.md [hunk 1]
```diff
@@ -5,9 +5,16 @@ Tag Monkeys is an open club dedicated to creating opportunities for people to be
...
 
 
 ## Join Tag Monkeys
 
-Do you want to help TM make our chapter colourful? Join the Slack channel #tagmonkeys in [METAs Slack](meta-lokal.slack.com). Do you just want to hang out and be creative on events? No membership is needed, come join us whenever!
+Do you want to help TM make our chapter colourful? Join the Tag Monkey's [Discord](https://discord.gg/yD3amRVmMT). Do you just want to hang out and be creative on events? No membership is needed, come join us whenever!
+
+
+## Tag Monkeys also has...
+Slack: The slack channel #tagmonkeys i [META-slacken](meta-lokal.slack.com)
```
Comment (niklasvatn) [inline en/clubs/tag-monkeys/body.md:17]:
```suggestion
Slack: The slack channel #tagmonkeys in [META-slacken](meta-lokal.slack.com)
```
nit

Comment (niklasvatn) [reply en/clubs/tag-monkeys/body.md:17]: or whatever preposition is correct.

**Review by niklasvatn — DISMISSED** [review]
Fix the things that I commented on if you want to :man_shrugging:

**Review by niklasvatn — APPROVED** [review]

**Review by Adasjo — APPROVED** [review]

**Pull request merged**